cosmos/loop.json

```diff
@@ -0,0 +1,55 @@
+{
+  "rpc": "https://rpc.loop.pfc.zone",
+  "rest": "https://api.loop.pfc.zone",
+  "nodeProvider": {
+    "name": "PFC",
+    "website":"https://pfc.zone/",
+    "email": "pfc@pfc.zone"
+  },
+  "chainId": "loop-1",
+  "chainName": "Loop Fans",
+  "stakeCurrency": {
+    "coinDenom": "TOKEN",
+    "coinMinimalDenom": "token",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/loop/token.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/loop",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/loop",
```

Comment:
The page isn't supported by Keplr Dashboard dynamically.
You can remove these two properties from the change.